### DISCUSSION TITLE
itsarraj

### DISCUSSION THREAD

**Discussion by ghost** [fardinkamal62/KOmegle#3] [Ideas]
itsarraj

**Comment by fardinkamal62** [comment]
That's great!
Where should we start?

**Comment by ghost** [comment]


**Reply by fardinkamal62** [reply]
If you're talking about auth, I guess we don't need it. It's supposed to be chat with random strangers

**Comment by ghost** [comment]


**Reply by fardinkamal62** [reply]
OK. What about creating password protected rooms?

**Reply by fardinkamal62** [reply]
okay you can try, fork this repo
